### DISCUSSION TITLE
Could not import PIL.Image

### DISCUSSION THREAD

**Discussion by defor27** [teamalgoritma/community#743] [Q&A] [answered]
Errror saat fitting model dengan keterangan berikut

Error in py_call_impl(callable, call_args$unnamed, call_args$named) : 
  ImportError: Could not import PIL.Image. The use of `load_img` requires PIL.

![image](https://github.com/teamalgoritma/community/assets/130421042/fc23ee7d-991d-4d4b-8d14-195fa22f1f91)

**Comment by defor27** [comment]
padahal untuk library pillow nya sudah terinstall

![image](https://github.com/teamalgoritma/community/assets/130421042/d2f96417-e89e-4795-a3d5-3fe12a40a13b)

**Reply by ramdhinafinita** [reply, marked as answer]
Dear @defor27,
Dari segi error yang disampaikan hal ini dikarenakan  "could not import PIL.Image," menunjukkan bahwa ada masalah dalam mengimpor modul PIL.Image yang dibutuhkan oleh fungsi load_img dari library keras (yang digunakan oleh Keras melalui TensorFlow).

Saran yang bisa dicoba:

1. Uninstall Pillow (PIL): `pip uninstall Pillow`
2. Install ulang Pillow (PIL): `pip install Pillow`
3. Restart kernel -> jalankan lagi chunk nya

![image](https://github.com/teamalgoritma/community/assets/64061969/32d6a672-0fc6-4c50-8b12-305652e5ac53)
Pillow yang terinstall dari bawaan anaconda bukan dari PyPi seperti library lain yang terinstall diatas. Mungkin bisa dicoba terlebih dahulu langkah-langkah diatas. Terkadang, ada versi library lain yang memiliki nama yang serupa, dan ini dapat menyebabkan konflik.

**Comment by defor27** [comment]
baik sudah bisa, terima kasih kak
